src/script/main/app.js

```diff
@@ -42,8 +42,9 @@ import {t} from 'utils/LocalizerUtil';
 /* eslint-disable no-unused-vars */
 import globals from './globals';
 import auth from './auth';
-import {getWebsiteUrl} from '../externalRoute';
 /* eslint-enable no-unused-vars */
+import {getWebsiteUrl} from '../externalRoute';
+import {enableLogging} from '../util/LoggerUtil';
```

Comment:
I think you can move those outside of the `eslint` comment (since the variables are used)

Reply:
They are below "no-unused" :)